concordium-consensus/tests/scheduler/SchedulerTests/Helpers.hs

```diff
@@ -150,7 +150,7 @@ createTestBlockStateWithAccounts accounts =
     keys = Types.withIsAuthorizationsVersionForPV (Types.protocolVersion @pv) $ DummyData.dummyKeyCollection
     seedState = case Types.consensusVersionFor (Types.protocolVersion @pv) of
         Types.ConsensusV0 -> initialSeedStateV0 (Hash.hash "") 1_000
-        Types.ConsensusV1 -> initialSeedStateV1 (Hash.hash "")
+        Types.ConsensusV1 -> initialSeedStateV1 (Hash.hash "") 3_600_000
```

Comment:
Do the 1000 and 3600000 here have any meaning?

Reply:
Not really. They are the epoch length in slots and timestamp (in ms since UNIX epoch) for triggering the first epoch transition respectively. The actual values are not relevant to the tests.